### ISSUE TITLE
Page view height auto detection when in iFrame plugin

### ISSUE THREAD

**Issue by DanielBickley** [ether/ep_page_view#25]
I see in the notes this plugin has "Automatic detection of page length to create pages" which is cool. However, I've got Etherpad-lite embedded using an iFrame and the page view seems to be stuck at a min height of 1500px - creating a long scrollbar for my parent box at 500px tall. When I adjust for this in the CSS it shortens up, but there is still a scrollbar, and it appears the 'ace_inner' iframe of the page view plugin is still too tall. Any thoughts on how best to get this height auto detection to work when embedded inside an parent iframe? I'm assuming the overall embed of my pad is causing the trouble.

For example, in my screenshot, is there any way to get the page (gray area) to end at about line 12? Isn't that what the auto height detection should be doing?

![screen shot 2015-06-23 at 8 27 59 pm](https://cloud.githubusercontent.com/assets/12854415/8322197/74c738b8-19e6-11e5-8558-655f7b9806d7.png)

**Comment by JohnMcLear** [comment]
No thoughts on best way

And it appears to be behaving as expected.  The page break should not be on line 12.

**Comment by DanielBickley** [comment]
So is "page view" basically structured to be similar to an A4 sheet of paper? I guess I was hoping to be able to create something that had the same sleek look as "page view" plugin but more of an A5 (shorter and narrower) - not necessarily stop at line 12, just load a little shorter and then expand as needed. I guess I find the scrollbar with large blank area useless for my purposes - I just like the "page view" look. I'll toy with it and see what happens. I'll note here the results of my experimenting and close this as a non-issue. Thanks.

**Comment by JohnMcLear** [comment]
Yep, A4 style.  Yea feel free to keep toying, happy for contributions if you can help maintain
